bin/plot_krun_results

```diff
@@ -727,6 +873,11 @@ def create_cli_parser():
                         help=('Name of the PDF file to write to. If no file is '
                               'specified, charts will be displayed interactively '
                               'and not written to disk.'))
+    parser.add_argument('--wallclock-only',
```

Comment:
There is >100 lines of code to set up the CLI parser, do you want it all "compressed" in this next commit, or just the lines that have changed in this PR?